cumulusci.yml

```diff
@@ -134,6 +139,8 @@ flows:
                 task: update_admin_profile
                 options:
                     managed: True
+            7:
+                task: add_application_type_picklist_values
```

Comment:
Can either @jstvz or @jofsky clarify if/why we need to run this as part of the uninstall_packaged_incremental?

Reply:
We don't, as it's a separate task.